src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesController.java

```diff
@@ -115,7 +115,7 @@ public ResponseEntity<Object> updateSubmission(
             //      all validation checks should be run, i.e. if the user is calling the PUT end-point from the
             //      'check your answers' screen.
             if (isValidationEnabled && isValidationRequired(overseasEntitySubmissionDto)) {
-                var validationErrors = overseasEntitySubmissionDtoValidator.validate(
+                var validationErrors = overseasEntitySubmissionDtoValidator.validatePartial(
```

Comment:
also same here - we don't need to do the isValidationRequired check here, just call validatePartial if isValidationEnabled

Reply:
Also removed and removed the code it calls as it's no longer required and pushing the % coverage down.